src/NuGet.Core/NuGet.Packaging/Signing/Authoring/X509SignatureProvider.cs

```diff
@@ -136,7 +136,12 @@ private static PrimarySignature CreatePrimarySignature(CmsSigner cmsSigner, Sign
 
             try
             {
+#if IS_DESKTOP
                 cms.ComputeSignature(cmsSigner);
+#else
+                // In .NET Framework, this parameter is not used and a PIN prompt is always shown. In .NET Core, the silent flag needs to be set to false to show a PIN prompt.
+                cms.ComputeSignature(cmsSigner, false);
```

Comment:
Suggestion: add a parameter name to the second argument

Reply:
Thanks for reviewing! Fixed.